python/lsst/ap/association/diaPipe.py

```diff
@@ -261,7 +261,7 @@ class DiaPipelineConfig(pipeBase.PipelineTaskConfig,
     )
     doWriteAssociatedSources = pexConfig.Field(
         dtype=bool,
-        default=False,
+        default=True,
```

Comment:
Good point. I will reverse this commit on this branch and set the config in `ApPipe` instead, and will not merge the changes to this package.

Reply:
I discussed this change further with Eric, and he would like us to change the default setting rather than overriding in `ApPipe`. In that case, I will keep and plan to merge this commit.